scripts/ci/validate-packages.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+
+set -e
+
+# capture all existing packages in the content dir.
+pkg_dirs=$(ls -l "themes/default/content/registry/packages/" | tail -n +2 | awk '{print $9}')
+
+# validates that all packages also have related content directory present.
+ls -l "themes/default/data/registry/packages" | tail -n +2 | awk '{print $9}' |  while read line; do
+    # extract the name from the frontmatter
+    pkg=$(cat "themes/default/data/registry/packages/${line}" | yq ".name")
+    # check to see that a dir exists for the package
+    if [[ "$pkg_dirs" != *"$pkg"* ]]; then
+        echo "ERROR: no content files found for package, $pkg"
```

Comment:
Nice. One question though -- since we're here, and checking that the directory exists, what do you think about going a step further and also checking that both content files exist also -- and ideally that they contain the right data (e.g., the correct `layout`)?

Reply:
yeah I can add some extra checks for those too